bazel/grpc_deps.bzl

```diff
@@ -55,21 +55,6 @@ def grpc_deps():
         actual = "@upb//:generated_code_support__only_for_generated_code_do_not_use__i_give_permission_to_break_me",
     )
 
-    native.bind(
-        name = "absl",
```

Comment:
Please double check that this doesn't require a cherrypick.

Reply:
Yep I just tested!